subsys/bluetooth/audio/vcp_vol_rend.c

```diff
@@ -192,9 +192,11 @@ static ssize_t write_vcs_control(struct bt_conn *conn,
 	if (volume_change && !vol_rend.flags) {
 		vol_rend.flags = 1;
 
-		bt_gatt_notify_uuid(NULL, BT_UUID_VCS_FLAGS,
-				    vol_rend.service_p->attrs,
-				    &vol_rend.flags, sizeof(vol_rend.flags));
+		if (IS_ENABLED(CONFIG_BT_VCP_VOL_REND_VOL_FLAGS_NOTIFIABLE)) {
+			bt_gatt_notify_uuid(NULL, BT_UUID_VCS_FLAGS,
+						vol_rend.service_p->attrs,
+						&vol_rend.flags, sizeof(vol_rend.flags));
```

Comment:
Indentation seems off here :) 

Should be

```suggestion
			bt_gatt_notify_uuid(NULL, BT_UUID_VCS_FLAGS,
					    vol_rend.service_p->attrs,
					    &vol_rend.flags, sizeof(vol_rend.flags));
```